ee/tasks/subscriptions/subscription_utils.py

```diff
@@ -31,7 +31,7 @@ def generate_assets(
 ) -> tuple[list[Insight], list[ExportedAsset]]:
     with SUBSCRIPTION_ASSET_GENERATION_TIMER.time():
         if resource.dashboard:
-            tiles = get_tiles_ordered_by_position(resource.dashboard)
+            tiles = get_tiles_ordered_by_position(resource.dashboard, "sm")
```

Comment:
I have mixed feelings about passing in `sm`, the `get_tiles_ordered_by_position` function is only used in one place, so maybe we should just default to `sm` instead of passing in as a param.

Let me know if yall got any preferences!

Reply:
Yes you're right, lets leave it as such for now. We eventually want to get the selected layout from the UI back to the backend but don't have a way to do that yet